css/index.css

```diff
@@ -112,6 +140,46 @@ p {
     justify-content: center;
 }
 
```

Comment:
great job utilizing a second class to target these. you may be able to achieve the same result with just calling the secondary class as such... .box1 { background: teal} instead of... .boxes .box.box1 {background: teal}